modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc

```diff
@@ -65,6 +65,21 @@ df.write.mode("overwrite").format("jdbc").options(
 
 The `sep` and `eol` will be used to convert the `DataFrame` to delimited data and that will be sent to the filename defined by the loading job.
 
+=== 2) Create a Loading Job
+
+Write a GSQL loading job, using the job and file names that you used in step 1, to map data from the CSV file(s) to TigerGraph vertices and edges.
+
+.Example:
+[source, gsql]
+CREATE LOADING JOB load_Social FOR GRAPH Social {
+    DEFINE FILENAME file1;
+    DEFINE FILENAME file2;
+    LOAD file1 TO VERTEX Person VALUES ($0, $1, $2);
+    LOAD file2 TO EDGE Friendship VALUES ($0, $1);
+}
+
+The loading job above, `load_Social` loads the 1st, 2nd, and 3rd columns of source file, `file1`, to the 1st, 2nd, and 3rd attributes of the vertex `Person`.
+
 .Example: DataFrame
```

Comment:
The examples of the DataFrame (input) and the Delimited Data should go with Step 1.

Reply:
Thank you. These have been moved with latest commit.